UIforETW/UIforETWDlg.cpp

```diff
@@ -282,7 +280,7 @@ BOOL CUIforETWDlg::OnInitDialog()
 		}
 	}
 
-	if (GetWindowsVersion() == kWindowsVersionXP)
+	if (!IsWindowsVistaOrGreater())

```

Comment:
Is there a WindowsXPOrLesser() function? Ditto for Windows7OrLesser()? The negatives in all of these expressions make the code harder to read.